### PR TITLE
New major version

### PR DESCRIPTION
**Pull request by thompsnm**
This pull request addresses all issues in the 'Publish 2.0.0' milestone and allows the use of either Protractor 1.x or 2.x in conjunction with CukeFarm. It does require a minimum of Cucumber 0.5.x to work, which is why it will be published as a new major version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +17,15 @@
...
     "coffee-script": "^1.9.1",
-    "docha": "thompsnm/docha#36f8d21",
+    "cucumber": "^0.5.0",
+    "docha": "^0.1.2",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-coffeelint": "^0.0.13",
     "grunt-contrib-coffee": "^0.13.0",
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-requirejs": "^0.4.4",
```
Comment (dansomething) [inline package.json]: @thompsnm is requirejs used anywhere in this project?

**Comment by dansomething** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (thompsnm) [inline package.json]: Yep, it's used to load `js-fixtures` into the test app that the integration tests run against. See [here](https://github.com/ReadyTalk/cukefarm/blob/master/spec/test_app/scripts/app.js#L1). It's arguably a heavyweight solution for a lightweight problem, but that way all dependencies in the project are managed through npm.

Comment (dansomething) [inline package.json]: So they only need to be loaded into the browser? And I guess they're not exported in a way that works with out requirejs?

Comment (thompsnm) [inline package.json]: As luck would have it, it's actually very easy to serve up js-fixtures using just express without requirejs. When you have a hammer, amirite?

**Comment by dansomething** [comment]
Nice work!

**Pull request merged**